src/utils/utils.py

```diff
@@ -80,3 +82,18 @@ def setup_logger():
         ],
     )
     return LOGGER
+
+
+def log_to_discord(scraper_outcomes: list[tuple[str, str]]):
+    url = "https://discord.com/api/webhooks/948021922661281883/IT9C3Q1ITRH4A_VUdfz8bYLLDum_qSY0c_BcNUlLebIgW1SLBKkxW5CZFNzfVvMQK5a0"
```

Comment:
Replace with an env var for sensitive information and use SAM to deploy secrets to lambda

Reply:
Using a secrets file for now.